File: Universal Windows Platform Solution/Phantom of the West/Phantom of the West/Visual Novel/Chapters/Chapter 5/IWouldDisarm.cs
using Phantom_of_the_West.Visual_Novel.Serialization.Event_Flagging;

namespace Phantom_of_the_West.Visual_Novel.Chapters.Chapter_5
{
	internal class IWouldDisarm : StoryChoice
	{
		internal IWouldDisarm() : base(StoryChoice.GetLocalizedString("ch5IWouldDisarm"), 5064)
		{
		}

		public override void Select()
		{
			PotWVN vn = PotWVN.MainVN;
			vn.EventFlags.Ch5AnsweredCoach = PhysicalEducationAnswer.Disarm;
			base.Select();
		}
	}
}

File: Android Project/app/src/main/java/com/christopherraleigh/phantomofthewest/visual_novel/chapters/chapter_5/IWouldDisarm.java
package com.christopherraleigh.phantomofthewest.visual_novel.chapters.chapter_5;

import com.christopherraleigh.phantomofthewest.R;
import com.christopherraleigh.phantomofthewest.visual_novel.PotWVN;
import com.christopherraleigh.phantomofthewest.visual_novel.chapters.StoryChoice;
import com.christopherraleigh.phantomofthewest.visual_novel.serialization.event_flagging.PhysicalEducationAnswer;

/**
 * Created by christopherraleigh on 2016-05-18.
 */
public class IWouldDisarm extends StoryChoice {
    public IWouldDisarm() {
        super(R.string.ch5IWouldDisarm, 5064);
    }

    @Override
    public void select() {
        PotWVN vn = PotWVN.getMainVN();
        vn.getEventFlags().setCh5AnsweredCoach(PhysicalEducationAnswer.Disarm);
        super.select();
    }
}
